qubesmanager/qube_manager.py

```diff
@@ -120,19 +120,26 @@ def __init__(self, vm, row_no, table):
         table.setItem(row_no, VmManagerWindow.columns_indices['Internal'],
                       self.internal_widget)
 
-        self.ip_widget = table_widgets.VmIPItem(vm)
+        self.ip_widget = table_widgets.VMPropertyItem(vm, "ip")
         table.setItem(row_no, VmManagerWindow.columns_indices['IP'],
                       self.ip_widget)
 
-        self.include_in_backups_widget = \
-            table_widgets.VmIncludeInBackupsItem(vm)
+        self.include_in_backups_widget = table_widgets.VMPropertyItem(
+            vm, "include_in_backups",
+            empty_function=(lambda x: not bool(x)))
         table.setItem(row_no, VmManagerWindow.columns_indices[
-            'Backups'], self.include_in_backups_widget)
+            'Include in backups'], self.include_in_backups_widget)
 
-        self.last_backup_widget = table_widgets.VmLastBackupItem(vm)
+        self.last_backup_widget = table_widgets.VmLastBackupItem(
+            vm, "backup_timestamp")
         table.setItem(row_no, VmManagerWindow.columns_indices[
             'Last backup'], self.last_backup_widget)
 
+        self.dvm_template_widget = table_widgets.VMPropertyItem(
+            vm, "template_for_dispvms", empty_function=(lambda x: not x))
+        table.setItem(row_no, VmManagerWindow.columns_indices['DVM Template'],
+                      self.dvm_template_widget)
```

Comment:
I think QubesOS/qubes-issues#4912 is really about (badly named) `default_dispvm` property. Which should really be called `default_dvm_template` or such...